### DISCUSSION TITLE
Wiki has incorrect arguments for set_auto_reconnect()

### DISCUSSION THREAD

**Discussion by z-l-p** [pschatzmann/ESP32-A2DP#555] [Q&A] [answered]
Hi. I just noticed that the explanation of **set_auto_reconnect()** shown on [https://github.com/pschatzmann/ESP32-A2DP/wiki/Auto-Reconnect](https://github.com/pschatzmann/ESP32-A2DP/wiki/Auto-Reconnect) shows 3 arguments, but there are only 2 expected. It looks like the middle argument (afterNormalDisconnect) is no longer used, so the example won't compile.
Thanks for making such inspiring libraries!

**Comment by pschatzmann** [comment, marked as answer]
The Example has been corrected
